packages/common/src/types/ApiMain.ts

```diff
@@ -0,0 +1,43 @@
+import z from 'zod'
+
+import { AssetId } from './AssetId'
+import { branded } from './branded'
+import { UnixTime } from './UnixTime'
+
+const ChartPoint = z.tuple([
+  branded(z.number(), (n) => new UnixTime(n)),
+  z.number(),
+  z.number(),
+])
+export type ChartPoint = z.infer<typeof ChartPoint>
+
+const Chart = z.object({
+  types: z.tuple([z.literal('timestamp'), z.string(), z.string()]),
+  data: z.array(ChartPoint),
+})
+export type Chart = z.infer<typeof Chart>
+
+export const Charts = z.object({
+  hourly: Chart,
+  qh6: Chart,
+  daily: Chart,
```

Comment:
Maybe `everyHour`, `every6Hours` and `everyDay`?

Reply:
Or even `sixHourly` which is maybe not fully grammatically correct but self explanatory

Reply:
`sixHourly` sounds good to me